src/plugins/intel_npu/src/backend/include/zero_types.hpp

```diff
@@ -13,10 +13,6 @@
 
 #include "intel_npu/config/runtime.hpp"
 
-/**
- * @brief Last version of Table of Graph Extension functions used within plugin
- */
-using ze_graph_dditable_ext_last_t = ze_graph_dditable_ext_1_8_t;
```

Comment:
Just a quick question here. Are these changes only for the graph dditable? I mean, the command queue structure also has the name 1_0.

Reply:
Just for ddi table, in https://github.com/intel/level-zero-npu-extensions/commit/a6487cc2c5da9aa13db9e005a320a1b6a0ee5919, just graph ext is impacted, other table is on level zero side.